services/datcore-adapter/src/simcore_service_datcore_adapter/core/settings.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+
+from models_library.basic_types import BootModeEnum, BuildTargetEnum, LogLevel
+from pydantic import Field
+from pydantic.networks import AnyUrl
+from settings_library.base import BaseCustomSettings
+from settings_library.logging_utils import MixinLoggingSettings
+
+
+class PennsieveSettings(BaseCustomSettings):
```

Comment:
The idea in the new strategy is that the field names are the same as the envs(even thought pydantic allows some tricks to avoid that constraint). This way it is easier to identify which variable corresponds to which field. Therefore, here it is convenient that you call it e.g.

```python
PENNSIEVE_ENABLED: bool = True
PENNSIEVE_API_URL: AnyUrl = "https://api.pennsieve.io"
```